ConwaysGameOfLife/Rules.cs

```diff
@@ -8,6 +10,13 @@ public class Rules
         // * Any live cell with more than three live neighbours dies, as if by overcrowding.
         // * Any live cell with two or three live neighbours lives on to the next generation.
         // * Any dead cell with exactly three live neighbours becomes a live cell.
-        
+        public Rules(Universe universe)

```

Comment:
Which way should this dependency be? We're currently injecting a `Universe` into an instance of `Rules`